src/substrait/textplan/SymbolTable.h

```diff
@@ -145,8 +148,13 @@ class SymbolTable {
       const std::any& subtype,
       const std::any& blob);
 
-  // Changes the location for a specified existing symbol.
-  void updateLocation(const SymbolInfo& symbol, const Location& location);
+  // Changes the permanent location for a specified existing symbol.
+  void addPermanentLocation(const SymbolInfo& symbol, const Location& location);
```

Comment:
Maybe expand this comment slightly to explain how "permanent" is different from source?

Reply:
Done